### ISSUE TITLE
Double triangle thingy when opening milestones editor

### ISSUE THREAD

**Issue by miketaylr** [webcompat/webcompat.com#1790]
<img width="127" alt="screen shot 2017-09-07 at 3 59 41 pm" src="https://user-images.githubusercontent.com/67283/30152414-9ac9de6e-93e5-11e7-8ac5-1b45874ca13d.png">

**Comment by miketaylr** [comment]
Aha, the category editor above (labels) is getting its state messed with as well:

```
<button class="wc-CategoryEditorLauncher r-ResetButton wc-Icon wc-Icon--gear js-CategoryEditorLauncher js-LabelEditorLauncher is-active" aria-hidden="true" style="display: inline-block;">
        <span class="wc-Accessibility">Edit Labels</span></button>
```

It shouldn't have `is-active`...

**Comment by miketaylr** [comment]
I think the bug might be here: 

https://github.com/webcompat/webcompat.com/blob/dddfd9b68d0582c1fdcf2efae516416c3a2d61c9/webcompat/static/js/lib/editor.js#L55-L61

**Comment by MDTsai** [comment]
Using correct EditorLauncher in fetchItems and editItems could fix part of this problem, but not reRender above.

**Comment by miketaylr** [comment]
Fixed by #1804.
